### PR TITLE
Blog Post: GraphQL Armor Support for GraphQL Yoga

### PR DESCRIPTION
**Pull request by glimow**
Adding the blog post to announce [GraphQL Armor](https://github.com/Escape-Technologies/graphql-armor)'s support for GraphQL Yoga 2

### CODE REVIEW

**Review by Urigo — CHANGES_REQUESTED** [review]

**Comment by notrab** [comment]
@n1ru4l @Urigo we also have [this video](https://graphql.wtf/episodes/55-graphql-armor) showing Armor + Yoga if we want to embed it.

```md
<YouTube youTubeId="Rklux1aF5Tc" />
```

File: pages/blog/announcing-graphql-armor-support-for-graphql-yoga-2.mdx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  const server = createServer({
+    schema,
+    plugins: [...enhancements.plugins],
+  });
+  return server;
+}
+```
+
+This example can be found in our example repository [github.com/dotansimha/graphql-yoga](https://github.com/dotansimha/graphql-yoga/tree/master/examples)
+
```
Comment (notrab) [inline pages/blog/announcing-graphql-armor-support-for-graphql-yoga-2.mdx:89]:
```suggestion

<YouTube youTubeId="Rklux1aF5Tc" />
```

**Comment by iCarossio** [comment]
Yes that’s definitely an excellent idea!

File: pages/blog/announcing-graphql-armor-support-for-graphql-yoga-2.mdx [hunk 2]
```diff
@@ -0,0 +1,101 @@
+import withArticle from '../../ui/blog/article'
+
+export const meta = {
+  title: 'Improved Security with GraphQL Armor support for Yoga Server 2',
```
[the remark this answers is not in the document]
Comment (charlypoly) [reply pages/blog/announcing-graphql-armor-support-for-graphql-yoga-2.mdx:4]: @n1ru4l ping me when done so I update the newsletter link 😅

**Comment by glimow** [comment]
@n1ru4l @Urigo updated article title and cover to add both our companies logo.

File: pages/blog/improved-security-with-graphql-armor-support-for-yoga-server-2.mdx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+## How does it look like to use GraphQL Armor with Yoga?
+
+GraphQL Armor relies on Envelop plugins for its security rules.
+
+Getting started is dead-simple: `npm install -S graphql-armor` (or `yarn add graphql-armor`)
+
+Then let's take a minimalistic Yoga server:
+
+```javascript=
```
Comment (n1ru4l) [inline pages/blog/improved-security-with-graphql-armor-support-for-yoga-server-2.mdx:55]:
```suggestion
```ts
```

File: pages/blog/improved-security-with-graphql-armor-support-for-yoga-server-2.mdx [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+  const server = createServer({
+    schema,
+  });
+  return server;
+}
+```
+
+Adding GraphQL armor is just a matter of adding a few envelop plugins:
+
+```javascript=
```
Comment (n1ru4l) [inline pages/blog/improved-security-with-graphql-armor-support-for-yoga-server-2.mdx:70]:
```suggestion
```ts
```

**Pull request merged**